### PR TITLE
remove unneeded parameters from transformTestFixture function

### PR DESCRIPTION
**Pull request by scitesy**

### CODE REVIEW

File: docs/compiling.md [hunk 1]
```diff
@@ -205,8 +205,7 @@ the `fable` command in the sample directory. Just be sure to install the npm dep
 the first time.
 
 ```shell
-npm install
-fable
+npm install -g fable-compiler
```
Comment (alfonsogarciacaro) [inline docs/compiling.md]: This was assuming Fable was already installed globally (as written at the beginning of the document): `npm install` here is to install the local dependencies of each sample and `fable` to compile the sample with the options from `fableconfig.json` (actually, right now only the second instruction would be necessary as `npm install` is included as a `rebuild` event in `fableconfig.json`).

Could you please remove this before merging the PR? Thanks!

**Comment by alfonsogarciacaro** [comment]
Nice one! This will resolve the warning when compiling the Test plugin, I've been too lazy to do that ;)

I've added a comment, could you please check it. Thanks!

File: docs/compiling.md (hunk 1 above)
Comment (scitesy) [inline docs/compiling.md]: Thanks for the review! I'll do this later today.

**Comment by alfonsogarciacaro** [comment]
Thanks for that!

**Pull request merged**